tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -0,0 +1,310 @@
+                IsGitShallowCloneSupported = true,
+                PathToUnitTestProject = "src/Cake.Common.Tests",
+                SupportedFrameworks = new[] { TargetFramework.NETCOREAPP3_1, TargetFramework.NET5_0, TargetFramework.NET6_0 },
+            },
+            ExplorationTestName.swashbuckle => new ExplorationTestDescription()
+            {
+                Name = "Swashbuckle.AspNetCore",
+                GitRepositoryUrl = "https://github.com/domaindrivendev/Swashbuckle.AspNetCore.git",
+                GitRepositoryTag = "v6.2.3",
+                IsGitShallowCloneSupported = true,
+                PathToUnitTestProject = "test/Swashbuckle.AspNetCore.SwaggerGen.Test",
+                SupportedFrameworks = new[] { TargetFramework.NET6_0 },
+            },
+            ExplorationTestName.paket => new ExplorationTestDescription()
+            {
+                Name = "Paket",
+                GitRepositoryUrl = "https://github.com/fsprojects/Paket.git",
+                GitRepositoryTag = "6.2.1",
+                IsGitShallowCloneSupported = true,
+                PathToUnitTestProject = "tests/Paket.Tests",
+                TestsToIgnore = new[] { "Loading assembly metadata works" },
+                SupportedFrameworks = new[] { TargetFramework.NET461, TargetFramework.NETCOREAPP3_1 },
+            },
+            ExplorationTestName.ilspy => new ExplorationTestDescription()
+            {
+                Name = "ILSpy",
+                GitRepositoryUrl = "https://github.com/icsharpcode/ILSpy.git",
+                GitRepositoryTag = "v7.1",
+                IsGitSubmodulesRequired = true,
+                PathToUnitTestProject = "ICSharpCode.Decompiler.Tests",
+                IsTestedByVSTest = true,
+                TestsToIgnore = new[] { "UseMc", "_net45", "ImplicitConversions", "ExplicitConversions", "ICSharpCode_Decompiler", "NewtonsoftJson_pcl_debug", "NRefactory_CSharp", "Random_TestCase_1", "AsyncForeach", "AsyncStreams", "AsyncUsing", "CS9_ExtensionGetEnumerator", "IndexRangeTest", "InterfaceTests", "UsingVariables" },
+                SupportedFrameworks = new[] { TargetFramework.NET461 },
+            },
+            _ => throw new ArgumentOutOfRangeException(nameof(name), name, null)
+        };
+
+        return description;
+    }
+}
+
+partial class Build
+{
+    AbsolutePath ExplorationTestsDirectory => RootDirectory / "exploration-tests";
+
+    [Parameter("Indicates name of exploration test to run. If not specified, will run all tests sequentially.")]
+    readonly ExplorationTestName? ExplorationTestName;
+
+    [Parameter("Indicates whether exploration tests should skip repository cloning. Useful for local development. Default false.")]
+    readonly bool ExplorationTestSkipClone;
+
+    [Parameter("Indicates whether exploration tests should run on latest repository commit. Useful if you want to update tested repositories to the latest tags. Default false.", 
+               List = false)]
```

Comment:
What does `List = false` control? Do we need it?

Reply:
`List = false` indicates, that while running `nuke --help` this parameter should not be listed.
I think hiding this parameter is good for 2 reasons:

1. Since `ExplorationTestCloneLatest` will not be used commonly with developers, it could confuse them, having this in the list.
2. Hiding this parameter will help reducing big list of all arguments.

But not mandatory, can remove flag if it bothers.

Reply:
oooh, I didn't know about that one. We should probably do that with some other parameters that generally don't need to be used locally!